_licenses/eupl-1.1.txt

```diff
@@ -0,0 +1,346 @@
+﻿---
+title: European Union Public License 1.1
+nickname: EUPL-1.1
+tab-slug: eupl-v1.1
+redirect_from: /licenses/eupl-v1.1/
+family: EUPL
+featured: false
+source: https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence-eupl-v11
+
+description: The “European Union Public Licence” (EUPL) The EUPL is the first European Free/Open Source Software (F/OSS) licence. It has been created on the initiative of the European Commission. It is now approved by the European Commission in 22 official languages of the European Union.
```

Comment:
"The EUPL" seems like a re-start of the sentence and could be removed. I don't think the "first" statement is true: CeCILL existed since 2004.